.circleci/config.yml

```diff
@@ -8,7 +11,7 @@ anchors:
 parameters:
   build_image_tag_ubuntu:
     type: string
-    default: slack-desktop-docker.jfrog.io/desktop-base-ubuntu-jammy:b2c26f2d-2038
```

Comment:
I'll see if I can bump our Ubuntu 22.04 image to also use Node 22, it should be pretty straightforward after bumping the Ubuntu 20.04 image - I just need to make sure it doesn't break the Ubuntu remote dev envs 👀

Reply:
(Does not block this PR, but just so we avoid being surprised in the future 😄 )